build/Dockerfile.debian12

```diff
@@ -6,6 +6,7 @@
 
 FROM python:3.8-slim-bookworm
 
+LABEL org.opencontainers.image.description "Debian 12 based image to run indy-node. See https://github.com/hyperledger/indy-node-container"
```

Comment:
```suggestion
LABEL org.opencontainers.image.description="Debian 12 based image to run indy-node. See https://github.com/hyperledger/indy-node-container"
```

Reply:
actually the container should get labeld in the work flow, but this seems not to be working.